### PR TITLE
zh-translation: docs/root/intro/arch_overview/other_protocols/grpc.rst

### PR DESCRIPTION
**Pull request by scilla0531**
ref: https://github.com/cloudnativeto/envoy/issues/73

For an explanation of how to fill out the fields, please see the relevant section
in [PULL_REQUESTS.md](https://github.com/envoyproxy/envoy/blob/master/PULL_REQUESTS.md)

Commit Message:
Additional Description:
Risk Level:
Testing:
Docs Changes:
Release Notes:
[Optional Runtime guard:]
[Optional Fixes #Issue]
[Optional Deprecated:]

### CODE REVIEW

File: docs/root/intro/arch_overview/other_protocols/grpc.rst [hunk 1]
```diff
@@ -3,78 +3,35 @@
 gRPC
 ====
 
-`gRPC <https://www.grpc.io/>`_ is an RPC framework from Google. It uses protocol buffers as the
-underlying serialization/IDL format. At the transport layer it uses HTTP/2 for request/response
-multiplexing. Envoy has first class support for gRPC both at the transport layer as well as at the
-application layer:
+`gRPC <https://www.grpc.io/>`_ 是来自 Google 的 RPC 框架。它使用协议缓冲区作为底层序列化/IDL（接口描述语言）格式。在传输层，它使用 HTTP/2 进行请求/响应复用。Envoy在传输层和应用层都提供对 gRPC 一流的支持：
```
Comment (shalk) [inline docs/root/intro/arch_overview/other_protocols/grpc.rst]: protocol buffers 这个不用翻译，专有名词

File: docs/root/intro/arch_overview/other_protocols/grpc.rst [hunk 2]
```diff
@@ -3,78 +3,35 @@
...
+Envoy 支持两种 gRPC 桥接过滤器:
 
-* :ref:`grpc_http1_bridge filter <config_http_filters_grpc_bridge>` which allows gRPC requests to be sent to Envoy over
-  HTTP/1.1. Envoy then translates the requests to HTTP/2 for transport to the target server. The response is translated back to HTTP/1.1.
-  When installed, the bridge filter gathers per RPC statistics in addition to the standard array of global HTTP statistics.
-* :ref:`grpc_http1_reverse_bridge filter <config_http_filters_grpc_http1_reverse_bridge>` which allows gRPC requests to be sent to Envoy
-  and then translated to HTTP/1.1 when sent to the upstream. The response is then converted back into gRPC when sent to the downstream.
-  This filter can also optionally manage the gRPC frame header, allowing the upstream to not have to be gRPC aware at all.
+* :ref:`gRPC HTTP/1.1 桥接过滤器 <config_http_filters_grpc_bridge>` 允许将 gRPC 请求通过 HTTP/1.1 发送到 Envoy。然后，Envoy 将请求转换为 HTTP/2 传输到目标服务器。响应则被转换回 HTTP/1.1。安装后，桥接过滤器除了收集标准的全局 HTTP 统计数据外，还收集每个 RPC 的统计数据。
+* :ref:`gRPC HTTP/1.1 反向桥接过滤器 <config_http_filters_grpc_http1_reverse_bridge>` 允许将 gRPC 请求发送到 Envoy，然后在发送到上游时转换为 HTTP/1.1。在发送到下游时，响应又被转换回 gRPC。该过滤器还可以选择管理 gRPC 帧头，这样一来，上游就完全不用知道 gRPC 了。
```
Comment (shalk) [inline docs/root/intro/arch_overview/other_protocols/grpc.rst]: 该过滤器还可以选择性地管理 gRPC 帧头

File: docs/root/intro/arch_overview/other_protocols/grpc.rst [hunk 3]
```diff
@@ -3,78 +3,35 @@
...
-gRPC services
+gRPC 服务
 -------------
 
-In addition to proxying gRPC on the data plane, Envoy makes use of gRPC for its
-control plane, where it :ref:`fetches configuration from management server(s)
-<config_overview>` and in filters, such as for :ref:`rate limiting
-<config_http_filters_rate_limit>` or authorization checks. We refer to these as
-*gRPC services*.
+Envoy 除了在数据平面上代理 gRPC 外，在控制平面上也使用了 gRPC，在控制平面上，它从中 :ref:`获取管理服务器的配置 <config_overview>` 以及过滤器的配置，例如将其用于 :ref:`限速 <config_http_filters_rate_limit>` 或授权检查。我们称之为 *gRPC 服务*。
```
Comment (majinghe) [inline docs/root/intro/arch_overview/other_protocols/grpc.rst]:
```suggestion
Envoy 除了在数据平面上代理 gRPC 外，在控制平面上也使用了 gRPC，在控制平面上，它从中 :ref:`获取管理服务器的配置 <config_overview>` 以及过滤器的配置，例如将其用于 :ref:`限流 <config_http_filters_rate_limit>` 或授权检查。我们称之为 *gRPC 服务*。
```

File: docs/root/intro/arch_overview/other_protocols/grpc.rst [hunk 4]
```diff
@@ -3,78 +3,35 @@
...
-breaking <arch_overview_circuit_break>`, :ref:`health checks
-<arch_overview_health_checking>`, :ref:`outlier detection
-<arch_overview_outlier_detection>`. They share the same :ref:`connection pooling
-<arch_overview_conn_pool>` mechanism as the Envoy data plane. Similarly, cluster
-:ref:`statistics <arch_overview_statistics>` are available for gRPC services.
-Since the client is minimal, it does not include advanced gRPC features such as
-`OAuth2 <https://oauth.net/2/>`_ or `gRPC-LB
-<https://grpc.io/blog/loadbalancing>`_ lookaside.
+Envoy gRPC 客户端是 gRPC 的最小自定义实现，它使用 Envoy 的 HTTP/2 上游连接管理。服务被指定为常规的 Envoy :ref:`集群 <arch_overview_cluster_manager>`，并定期处理 :ref:`超时、重试 <arch_overview_http_conn_man>`、终端 :ref:`发现 <arch_overview_dynamic_config_eds>`、:ref:`负载均衡、故障转移 <arch_overview_load_balancing>`、负载报告、:ref:`断路 <arch_overview_circuit_break>`、:ref:`健康检查 <arch_overview_health_checking>`、:ref:`异常检测 <arch_overview_outlier_detection>`。它们与 Envoy 的数据面共享相同的 :ref:`连接池 <arch_overview_conn_pool>` 机制。同样，集群 :ref:`统计信息 <arch_overview_statistics>` 用于 gRPC 服务。由于客户端是简化版的 gRPC 实现，因此不包括诸如 `OAuth2 <https://oauth.net/2/>`_ 或 `gRPC-LB <https://grpc.io/blog/loadbalancing>`_ 之类的高级 gRPC 功能后备。
```
Comment (majinghe) [inline docs/root/intro/arch_overview/other_protocols/grpc.rst]:
```suggestion
Envoy gRPC 客户端是 gRPC 的最小自定义实现，它使用 Envoy 的 HTTP/2 上游连接管理。服务被指定为常规的 Envoy :ref:`集群 <arch_overview_cluster_manager>`，并定期处理 :ref:`超时、重试 <arch_overview_http_conn_man>`、端点 :ref:`发现 <arch_overview_dynamic_config_eds>`、:ref:`负载均衡/故障转移 <arch_overview_load_balancing>`、负载报告、:ref:`熔断 <arch_overview_circuit_break>`、:ref:`健康检查 <arch_overview_health_checking>`、:ref:`异常检测 <arch_overview_outlier_detection>`。它们与 Envoy 的数据面共享相同的 :ref:`连接池 <arch_overview_conn_pool>` 机制。同样，集群 :ref:`统计信息 <arch_overview_statistics>` 用于 gRPC 服务。由于客户端是简化版的 gRPC 实现，因此不包括诸如 `OAuth2 <https://oauth.net/2/>`_ 或 `gRPC-LB <https://grpc.io/blog/loadbalancing>`_ 之类的高级 gRPC 功能后备。
```

**Comment by majinghe** [comment]
@scilla0531 加了点comment，仅供参考。

**Comment by scilla0531** [comment]
@majinghe  @shalk 感谢review，修改已经提交，麻烦查阅下哈！

**Pull request merged**